src/plugins/github/nodes.js

```diff
@@ -36,6 +37,24 @@ export const Prefix = deepFreeze({
   pullComment: _githubAddress(COMMENT_TYPE, PULL_TYPE),
 });
 
+/**
+ * Return the address corresponding to a GitHub login.
```

Comment:
GitHub logins can also correspond to organizations (they can correspond
to any `Actor`). Presumably we don’t care about those because we don’t
have nodes for organizations, but that probably merits at least a
comment?

Reply:
Done.